.github/workflows/pr.yaml

```diff
@@ -122,3 +139,18 @@ jobs:
         sccache -z;
         build-all -DBUILD_BENCHMARKS=ON --verbose;
         sccache -s;
+
+  telemetry-summarize:
+    runs-on: ubuntu-latest
+    needs: pr-builder
```

Comment:
pr-builder has always been the last job until now. PRs can be merged once that completes. Is it important for this to complete before merges occur?

Reply:
It was easy to piggy-back on pr-builder being the last thing so that everything else is done. It is not important for this to be complete before merges occur, but it is important that it start only after all other jobs (except pr-builder) are complete. Is there a better way to express that? Making it need conda-python-tests and wheel-python-tests would probably be good enough, and then this would run in parallel with pr-builder. I find that conceptually more messy, but I'm fine with it from a practical standpoint.

If the concern is that the merge would mess up the telemetry, I think that's also going to be OK. The telemetry is all based on github actions run metadata, and I believe that continues to exist until it ages out after some amount of time. The few seconds between a merge event and the summarize action won't matter.

Reply:
Great. I am fine with this depending on pr-builder.